docs/maintaining.md

```diff
@@ -160,22 +190,39 @@ curl -X PUT --header 'Content-Type: application/json' -d '
 
 ### Resetting your GraphDB instance
 
-If you previously set up a Neurobagel node on your machine but want to reset your graph database to start again _from scratch_, 
-the most foolproof way would be to start with a clean GraphDB configuration to avoid conflicts with any previously created credentials or databases.
+Each Neurobagel node has its own GraphDB instance, which is used to store the graph data for the node.
+If you want to reset your graph database and start again from scratch,
+follow these steps:
+
+1. Ensure that your Neurobagel node is not running (i.e., shut down the Docker containers for the node).
+
+    ```bash
+    docker compose --profile full_stack down
+    ```
+
+    If you are not using the `full_stack` profile, replace `full_stack` with the name of the profile you are using.
+
+2. Delete the docker volume that contains the GraphDB data for your node.
+
+    ```bash
+    docker volume rm neurobagel_graphdb_data
```

Comment:
To ensure we don't run into volume name clashes or repetition so that it is easy to identify which volume is associated with which deployment (e.g., recipes_graphdb_data, recipes_graphdb_data-1 gets confusing fast), here's an issue to document the `COMPOSE_PROFILE_NAME`:

- See https://github.com/neurobagel/documentation/issues/201

Reply:
```suggestion
    docker volume rm neurobagel_node_graphdb_home
```
same as above for graphdb_home, and `neurobagel_node` should be the default `COMPOSE_PROFILE_NAME`